dev/run-python-skinny-tests.sh

```diff
@@ -13,8 +13,9 @@ err=0
 trap 'err=1' ERR
 export MLFLOW_SKINNY='true'
 
-pytest --verbose tests/test_skinny.py
+pytest --verbose tests/test_skinny_sql.py
 python -m pip install sqlalchemy alembic sqlparse
+pytest --verbose tests/test_skinny.py
```

Comment:
Nit: Is it possible to provide slightly more descriptive names for these test suites? E.g.:

`test_skinny_client_omits_sql_libs`
`test_skinny_client_omits_data_science_libs`